### ISSUE TITLE
LG G5: No sound when playing ALAC audio track

### ISSUE THREAD

**Issue by gemiren** [google/ExoPlayer#5938]
### [REQUIRED] Content description
The m4a file loads in ExoPlayer without any error. The time bar even progresses normally when it plays but no sound comes out.

### [REQUIRED] Link to test content
http://feeds.soundcloud.com/stream/624159534-the-stemma-report-ep05.m4a

### [REQUIRED] Version of ExoPlayer being used
2.10.1

### [REQUIRED] Device(s) and version(s) of Android being used
Pixel 3 Android P

**Comment by marcbaechinger** [comment]
The m4a stream you posted does contain an ALAC file. The device doesn't have a decoder for this so the track can't be played with the default renderer. When I add the uri to the demo app it gives me the following output:

```
D/EventLogger: tracksChanged [2.90, 0.00, window=0, period=0, 
D/EventLogger:   Renderer:1 [
D/EventLogger:     Group:0, adaptive_supported=N/A [
D/EventLogger:       [ ] Track:0, id=1, mimeType=audio/alac, channels=2, sample_rate=44100, language=```, supported=NO_UNSUPPORTED_TYPE
D/EventLogger:     ]
D/EventLogger:   ]
D/EventLogger: ]
```

If you want to play files of that format with ExoPlayer you need to use the [ffmpeg extension](https://github.com/google/ExoPlayer/tree/release-v2/extensions/ffmpeg).

**Comment by gemiren** [comment]
Well, the android.media.MediaPlayer can play that m4a file without any issue. I'd expect ExoPlayer to play it without the ffmpeg extension since android.media.MediaPlayer can play it.

**Comment by marcbaechinger** [comment]
I did not manage to play the file on a Pixel 3 with P. Can you give me some details how you did this?

**Comment by gemiren** [comment]
I'm sorry. I should clarify that I was able to play that file using MediaPlayer on my LG G5 phone. However, ExoPlayer still fails to paly it on LG G5.

And you are right that it failed to play with both MediaPlayer and ExoPlayer on Pixel 3.

**Comment by ojw28** [comment]
Fixed in the `dev-v2` branch. The fix will be included in the 2.10.4 release.
